Delta/Emulation/GameViewController.swift

```diff
@@ -1070,6 +1087,71 @@ extension GameViewController
             emulatorCore.rate = emulatorCore.deltaCore.supportedRates.lowerBound
         }
     }
+    
+    func performStatusBarAction()
+    {
+        let enabled = ExperimentalFeatures.shared.showStatusBar.isStatusBarEnabled
+        ExperimentalFeatures.shared.showStatusBar.isStatusBarEnabled = !enabled
+        
+        if let pauseView = self.pauseViewController
+        {
+            pauseView.dismiss()
+        }
+        
+        self.setNeedsStatusBarAppearanceUpdate()
+    }
+    
+    func performScreenshotAction()
+    {
+        guard let snapshot = self.emulatorCore?.videoManager.snapshot() else { return }
+        
+        let scale = ExperimentalFeatures.shared.gameScreenshots.scale != nil ? ExperimentalFeatures.shared.gameScreenshots.scale?.rawValue : 1.0
+        let size = CGSize(width: snapshot.size.width * scale!, height: snapshot.size.height * scale!)
+        
+        UIGraphicsBeginImageContextWithOptions(size, false, 1.0)
+        UIGraphicsGetCurrentContext()!.interpolationQuality = .none
```

Comment:
`UIGraphicsBeginImageContextWithOptions` is a pretty old API, could you replace it with `UIGraphicsImageRenderer`? https://developer.apple.com/documentation/uikit/uigraphicsimagerenderer

Separately, try to keep force-unwrapping optionals to absolute minimum.